src/components/BarChart/WalletChart.tsx

```diff
@@ -22,32 +24,33 @@ type Props = {
 const WalletChart = ({ height }: Props) => {
   const dispatch = useAppDispatch()
   const {
-    account: { activityChart },
+    account: { activityChart, activityChartRange },
     activity: { filter },
   } = useSelector((state: RootState) => state)
```

Comment:
I think this change would eliminate renders happening every time the store is updated, but no actual change happens

``` javascript
  const {
    account: { activityChart, activityChartRange },
    activity: { filter },
  } = useSelector(
    (state: RootState) => state,
    (prev, next) => {
      const activityChartEqual = isEqual(
        prev.account.activityChart,
        next.account.activityChart,
      )
      const rangeEqual =
        prev.account.activityChartRange === next.account.activityChartRange
      const filterEqual = prev.activity.filter === next.activity.filter

      return activityChartEqual && rangeEqual && filterEqual
    },
  )
```

Reply:
ok thanks will play around with that and see if it improves it